apps/angular/crud/src/app/store/app.actions.ts

```diff
@@ -0,0 +1,53 @@
+import { createAction, props } from '@ngrx/store';
+// import { Action } from '@ngrx/store';
+import { TODO } from '../app.service';
+
+export const GETALLTODO = '[Todo] GetAllTodo';
+export const UPDATETODO = '[Todo] UpdateTodo';
+export const SUCCESSUPDATEDTODO = '[Todo] SuccessfullyUpdatedTodo';
+export const DELETETODO = '[Todo] DeleteTodo';
+export const SUCCESSDELETEDTODO = '[Todo] SuccessfullyDeletedTodo';
+export const ADDEDALLTODO = '[Todo] AddedAllTodo';
+export const ERRORACTION = '[Todo] Error Occured';
```

Comment:
That's very old school to create constant for each action. This was needed in the old ngrx store.